core/src/main/java/io/questdb/cutlass/text/types/SymbolAdapter.java

```diff
@@ -51,8 +54,18 @@ public boolean probe(CharSequence text) {
 
     @Override
     public void write(TableWriter.Row row, int column, DirectByteCharSequence value) throws Exception {
-        utf8Sink.clear();
+        StringSink utf8Sink = Misc.getThreadLocalBuilder();
```

Comment:
Thread Local is slightly slower than field access. On hot path it can yield some difference.

I'd change signature so that StringSink is passed to the method and then get reference to Thread Local outside of field / row loop once.